index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="scripts/main.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/mainStyle.css">
+    <title>Stephen's portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="toolbar">
+            <ul>
+                <li class="textPlaceholder"> <a href="#" aria-label="Ssenkooto Stephen"> Ssenkooto Stephen </a> </li>
+                <li> <button id="menuButton" type="button"> <img src="icons/menu.svg" alt=""> </button></li>
+                <li id="navigation">
+                    <ul class="container">
+                        <li class="textPlaceholder"><a href="#projects" aria-label="Portfolio"> Portfolio</a></li>
+                        <li class="textPlaceholder"><a href="#about" aria-label="About"> About</a></li>
+                        <li class="textPlaceholder"><a href="#contact" aria-label="Contact"> Contact</a></li>
+                        <li class="textPlaceholder"><a rel="noopener noreferrer" href="mailto:ssenkootos74@gmail.com"
+                                aria-label="Send Email"><img src="icons/mail.svg" alt="Send Email"></a></li>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section id="menu" class="menu-invisible">
+        <button type="button" id="closeButton"><img src="icons/close-icon.svg" alt="close icon"></button>
+        <ul class="container content menu">
+            <li class="textPlaceholder menu-item"><a href="#projects" aria-label="Portfolio"> Portfolio</a></li>
+            <li class="textPlaceholder menu-item"><a href="#about" aria-label="About"> About</a></li>
+            <li class="textPlaceholder menu-item"><a href="#contact" aria-label="Contact"> Contact</a></li>
+        </ul>
+    </section>
```

Comment:
Since this piece of code was not used. I think it would be better not to leave unused code in the codebase.